app/assets/javascripts/books.coffee

```diff
@@ -0,0 +1,36 @@
+$ ->
+  bsBooksModule = do ->
+    init: ->
+      $('.img-link').click (e) ->
```

Comment:
would be better if run all this code only on show book page 
add after `init: ->` something like:
`return unless $('show-book-page').length`

Reply:
and do it for all other js modules

Reply:
Actually not needed as the controller specific js approach was chosen:
1. removed coffees require links from [`application.js`](https://github.com/evtik/book_store/blob/c7855bb5c2afeaee80af83169a8b4f261f74bf83/app/assets/javascripts/application.js#L1)
2. in [`assets.rb`](https://github.com/evtik/book_store/blob/c7855bb5c2afeaee80af83169a8b4f261f74bf83/config/initializers/assets.rb#L9) each coffee is a sort of manually precompiled
3. and then in [`application.html.haml`](https://github.com/evtik/book_store/blob/c7855bb5c2afeaee80af83169a8b4f261f74bf83/app/views/layouts/application.html.haml#L10) only this controller specific js is included

Yeah, I'm loosing caching which I'd have in a case of an all-in-one js, but instead there's no need to add checks to js code like 'if there are particular elements on the page then add some their event handlers'.